auto_install/install.sh

```diff
@@ -1036,19 +1146,32 @@ chooseUser(){
 	whiptail --msgbox --backtitle "Parsing User List" --title "Local Users" "Choose a local user that will hold your ovpn configurations." ${r} ${c}
 	# First, let's check if there is a user available.
 	numUsers=$(awk -F':' 'BEGIN {count=0} $3>=1000 && $3<=60000 { count++ } END{ print count }' /etc/passwd)
-	if [ "$numUsers" -eq 0 ]
-	then
+	if [ "$numUsers" -eq 0 ]; then
 		# We don't have a user, let's ask to add one.
-		if userToAdd=$(whiptail --title "Choose A User" --inputbox "No non-root user account was found. Please type a new username." ${r} ${c} 3>&1 1>&2 2>&3)
-		then
+		if userToAdd=$(whiptail --title "Choose A User" --inputbox "No non-root user account was found. Please type a new username." ${r} ${c} 3>&1 1>&2 2>&3); then
 			# See https://askubuntu.com/a/667842/459815
 			PASSWORD=$(whiptail  --title "password dialog" --passwordbox "Please enter the new user password" ${r} ${c} 3>&1 1>&2 2>&3)
 			CRYPT=$(perl -e 'printf("%s\n", crypt($ARGV[0], "password"))' "${PASSWORD}")
-			if $SUDO useradd -m -p "${CRYPT}" -s /bin/bash "${userToAdd}" ; then
-				echo "Succeeded"
-				((numUsers+=1))
+
+			if [ "${PLAT}" == 'Alpine' ]; then
+				if ${SUDO} adduser -Ds /bin/bash "${userToAdd}"; then
+					${SUDO} addgroup "${userToAdd}" wheel
+
+					${SUDO} passwd "${userToAdd}" <<< "${CRYPT}"
```

Comment:
`chpasswd` works for setting the password

```suggestion
					${SUDO} chpasswd <<< "${userToAdd}":"${PASSWORD}"
```